mock/schedule.ts

```diff
@@ -15,7 +15,7 @@ export const Schedule: ISchedule = {
         borderColor: '#89609E',
         is_serviceSession: true,
         is_keynote: false,
-        is_bookmarked: false,
+        is_bookmarked: true,
```

Comment:
I don't know if having an `is_bookmarked` field here is the best play. I think we should use AsyncStorage to store the sessions a user has bookmarked. this is because different users will bookmark different sessions.

Reply:
oh, the schedule data response comes with a property `is_bookmarked`. I changed the value of some sessions just to mock sessions with that value as true and others false. There's an endpoint for bookmarking sessions.